sdk/attestation/Microsoft.Azure.Attestation/src/Customized/AttestationCredentials.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Rest;
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Attestation
+{
+    /// <summary>
+    /// The Attestation credential class that implements <see cref="ServiceClientCredentials"/>
+    /// </summary>
+    public class AttestationCredentials : ServiceClientCredentials
+    {
+        /// <summary>
+        /// Bearer token
+        /// </summary>
+        public string token { get; set; }
```

Comment:
`token` -> `Token` as it is a public API.

Reply:
Change to Token